nullaway/src/test/java/com/uber/nullaway/NullAwaySerializationTest.java

```diff
@@ -151,7 +148,42 @@ public void add_nullable_returnType_superClass() {
   }
 
   @Test
-  public void add_nullable_paramType_subclass() {
+  public void add_nullable_param_simple() {
+    serializationTestHelper
+        .setArgs(
+            Arrays.asList(
+                "-d",
+                temporaryFolder.getRoot().getAbsolutePath(),
+                "-XepOpt:NullAway:AnnotatedPackages=com.uber",
+                "-XepOpt:NullAway:SerializeFixMetadata=true",
+                "-XepOpt:NullAway:FixSerializationConfigPath=" + configPath))
+        .addSourceLines(
+            "com/uber/android/Super.java",
+            "package com.uber;",
+            "import javax.annotation.Nullable;",
+            "import javax.annotation.Nonnull;",
+            "public class Super {",
+            "   Object test(int i, Object h) {",
+            "     return h;",
+            "   }",
+            "   Object test_param(@Nullable String o) {",
+            "   // BUG: Diagnostic contains: passing @Nullable",
```

Comment:
Match indentation of the following line.

Reply:
b144824daae8571825ecbf1f1e72b4b81db23dca